gw_spaceheat/actors/scada_base.py

```diff
@@ -17,6 +17,7 @@ class ScadaBase(ActorBase):
     def __init__(self, node: ShNode, logging_on=False):
         super(ScadaBase, self).__init__(node=node, logging_on=logging_on)
         self.gwMqttBroker = settings.GW_MQTT_BROKER_ADDRESS
```

Comment:
Should we refactor the other `settings` calls with the getattr pattern?

Reply:
I think we should refactor settings to be an object you pass, by default read from env, but that’s means updating all users (you me and Pi) so I didn’t want to do it until we discus.

Reply:
Yes.